libs/core/langchain_core/utils/function_calling.py

```diff
@@ -295,14 +295,12 @@ def convert_to_openai_function(
         k in function for k in ("name", "description", "parameters")
     ):
         return function
-    # a JSON schema with title and description
-    elif isinstance(function, dict) and all(
-        k in function for k in ("title", "description", "properties")
-    ):
+    # a JSON schema
+    elif isinstance(function, dict) and "properties" in function:
         function = function.copy()
         return {
-            "name": function.pop("title"),
-            "description": function.pop("description"),
+            "name": function.pop("title", "extract"),
```

Comment:
This I am a bit opposed since this isn't default API behavior? I don't think we should be auto assigning a title